doc/dbus/org.opensuse.Agama.Storage1.Proposal.Calculator.doc.xml

```diff
@@ -18,8 +18,8 @@
       <!--
         MountPath               s
         MountOptions            as
-        TargetDevice            s
-        TargetVG                s
+        Target                  s
+        TargetDevice            s (only makes sense if Target is not default)
```

Comment:
The equivalent for the default device are called `Target` and `Device`. I am not sure if we should follow the same naming or not.

Reply:
I like the current names because it shows the relationship between both fields. I mean, `TargetDevice` is actually some kind of "details" field for the `Target` one.

Reply:
Ok, makes sense. Then, I will adapt my PR to follow the same approach.